README.rst

```diff
@@ -70,9 +70,99 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Windows** users:
 
-Note: You need to have bower installed before you may install the python packages.
+   - Clone the **coala-html** repository
```

Comment:
Should probably link to the repo here.

Reply:
The user will be reading the README.md which is at coala-html repo on GitHub. So, I don’t think so it is necessary to give a link here which will redirect user again to the same page.

Reply:
Ah didn't notice. Seems fine then.